honest_forests/estimators/forest.py

```diff
@@ -446,7 +445,6 @@ def _predict_proba(self, X, indices=None, impute_missing=None):
         Parallel(
             n_jobs=n_jobs,
             verbose=self.verbose,
-            **_joblib_parallel_args(require="sharedmem")
```

Comment:
I think this line should be `require="sharedmem"`

Reply:
sure. will change that.